doc/sphinx/zrefs.bib

```diff
@@ -438,24 +438,6 @@ @BOOK{hockney88
   timestamp = {2007.08.29}
 }
 
-@ARTICLE{hofling11a,
```

Comment:
Is there a reason for removing this entry?

Reply:
OK, offline discussion: This article isn't cited anywhere, which produces a warning. Thus removed.